systemsaholicVersion

```diff
@@ -0,0 +1,71 @@
+<?php
+/*
+ * Plugin Name: Sysaholic Contact Merge
+ * Plugin URI:
+ * Description: Merge back contact data between modules.
+ * Version: 1.0
+ * Author: Systemsaholic
+ * Author URI: http://www.systemsaholic.com
+ * Text Domain: portlight-groundhogg
+ * Domain Path: /languages
+ */
+
+
+//New ticket is created via HTTP POST Request to /wp-admin/admin.php From Twilio - Need to check if its in fact a SMS message
+if (!empty($_POST['action']) && strpos($_POST['request_email'], '@sms') !== false) {
```

Comment:
I think you will not receive email from Twilio SMS, only phone number.

Reply:
Correct, but WSDESK treats all incoming tickets as emails thus the POST calls on the request_email. I later stip the Phone number from that field.

Reply:
but where will $_POST['request_email'] come from? As I said, the Twilio seems to not send this field.